apps/omg/test/omg/state/transaction/recovered_test.exs

```diff
@@ -108,52 +109,81 @@ defmodule OMG.State.Transaction.RecoveredTest do
       assert {:error, :malformed_transaction} = Transaction.Recovered.recover_from(ExRLP.encode([sigs, []]))
 
       assert {:error, :malformed_transaction} =
-               Transaction.Recovered.recover_from(ExRLP.encode([sigs, inputs, outputs]))
+               Transaction.Recovered.recover_from(ExRLP.encode([sigs, inputs, outputs, 0, <<0::256>>]))
 
       assert {:error, :malformed_transaction} =
-               Transaction.Recovered.recover_from(ExRLP.encode([sigs, "bad_marker", inputs, outputs]))
+               Transaction.Recovered.recover_from(ExRLP.encode([sigs, "bad_marker", inputs, outputs, 0, <<0::256>>]))
 
       assert {:error, :malformed_witnesses} ==
-               Transaction.Recovered.recover_from(ExRLP.encode([[<<1>>, <<1>>], @payment_marker, inputs, outputs]))
+               Transaction.Recovered.recover_from(
+                 ExRLP.encode([[<<1>>, <<1>>], @payment_marker, inputs, outputs, 0, <<0::256>>])
+               )
 
       assert {:error, :malformed_witnesses} ==
-               Transaction.Recovered.recover_from(ExRLP.encode([<<1>>, @payment_marker, inputs, outputs]))
+               Transaction.Recovered.recover_from(
+                 ExRLP.encode([<<1>>, @payment_marker, inputs, outputs, 0, <<0::256>>])
```

Comment:
```suggestion
                 ExRLP.encode([1, @payment_marker, inputs, outputs, 0, <<0::256>>])
```

Just want to highlight we should start moving towards the transaction format's specification, which I think transaction type should be 1? https://github.com/omisego/plasma-contracts/blob/master/plasma_framework/docs/integration-docs/integration-doc.md#payment-transaction-rlp-encoding

Reply:
ah yes, good catch

Reply:
ah no, that's not the type, those are malformed witnesses, payment marker comes next